raptorWeb/staffapps/forms.py

```diff
@@ -0,0 +1,186 @@
+from django import forms
+from django.core import validators
+import logging
+
+from staffapps.models import ModeratorApplication, AdminApplication
+
+LOGGER = logging.Logger("form_validator_logger")
+
+FORM_LABELS = {
```

Comment:
This is extremely unclean. If keeping the form labels separate from the forms themselves is required, it may be better to place these into their own module, and call them as constants.

This is not conducive to a app where you can quickly create new forms, however. A larger re-write of this may be in order.